sentry-android-core/src/main/java/io/sentry/android/core/performance/ActivityLifecycleTimeSpan.java

```diff
@@ -0,0 +1,29 @@
+package io.sentry.android.core.performance;
+
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+@ApiStatus.Internal
+public class ActivityLifecycleTimeSpan implements Comparable<ActivityLifecycleTimeSpan> {
+  private final @NotNull TimeSpan onCreate = new TimeSpan();
+  private final @NotNull TimeSpan onStart = new TimeSpan();
+
```

Comment:
I'd be curious why don't we also have `onResume` span here? I think it's also happening before the first draw is done, or not always?

Reply:
Yes, we could add this one as well :+1:, let me follow up with an improvement PR here